qemu/tests/hotplug_virtio_mem.py

```diff
@@ -53,4 +55,15 @@ def run(test, params, env):
                                               threshold, vm, test)
         virtio_mem_utils.check_numa_plugged_mem(node_id, requested_size,
                                                 threshold, vm, test)
+    try:
+        hotplug_test.unplug_memory(vm, target_mem)
+    except QMPCmdError as e:
+        if error_msg not in str(e.data):
+            test.fail("Unexpected error message: %s" % str(e.data))
+        test.log.info(error_msg)
```

Comment:
Miss one failed scenario that unplug memory success, which is not unexpected.
suggest add:
```
else:
    test.fail(*****)
```

Reply:
Code updated !